qa/pull-tester/rpc-tests.py

```diff
@@ -376,10 +392,9 @@ def get_next(self):
             log_stderr = tempfile.SpooledTemporaryFile(max_size=2**16)
             self.jobs.append((t,
                               time.time(),
-                              subprocess.Popen((self.tests_dir + t).split() + self.flags + port_seed,
-                                               universal_newlines=True,
-                                               stdout=log_stdout,
-                                               stderr=log_stderr),
+                              self.start_test((self.tests_dir + t).split() + self.flags + port_seed,
```

Comment:
Pre-existing issue: consider using `os.path.join` instead of concatenating paths as strings.